pub/lib/dependabot/pub/update_checker.rb

```diff
@@ -2,12 +2,26 @@
 
 require "dependabot/update_checkers"
 require "dependabot/update_checkers/base"
+require "dependabot/pub/helpers"
 
 module Dependabot
   module Pub
     class UpdateChecker < Dependabot::UpdateCheckers::Base
       include Dependabot::Pub::Helpers
 
+      def initialize(dependency:, dependency_files:, repo_contents_path: nil,
+                     credentials:, ignored_versions: [],
+                     raise_on_ignored: false, security_advisories: [],
+                     requirements_update_strategy: nil,
+                     options: {}, test_host: nil)
+        @test_host = test_host
```

Comment:
Why don't we call this variable for `pub_hosted_url`?

Reply:
yeah - why not :)
Changed